tests/test_stackexchange.py

```diff
@@ -415,6 +415,22 @@ def request_callback(method, uri, headers):
         diff = after - before
         self.assertGreaterEqual(diff, 0.2)
 
+    def test_sanitize_for_archive(self):
+        """Test whether the sanitize method works properly"""
+
+        payload = {'order': 'desc',
+                   'site': 'stackoverflow',
+                   'sort': 'activity',
+                   'pagesize': 1,
+                   'key': 'aaa',
+                   'filter': 'Bf*y*ByQD_upZqozgU6lXL_62USGOoV3)MFNgiHqHpmO_Y-jHR',
+                   'page': 1,
+                   'tagged': 'python'}
+
+        url, headers, payload = StackExchangeClient.sanitize_for_archive(None, None, payload)
```

Comment:
Instead of using a `None` value for `url` and `headers`, I suggest to check it when these parameters have values too. Thus, you can be sure that both values are not modified by the method.

This comment is applied to the rest of the backends which override this method.